### ISSUE TITLE
debugger fails to show context for byte-compiled functions

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#6526]
### System details

    RStudio Edition : Desktop
    RStudio Version : v1.3.926
    OS Version      : macOS 10.15.3
    R Version       : 3.6.1

### Steps to reproduce the problem

Create a new project, with the following scripts:

### script.R

```
source("tools.R")
action(mtcars)
action(1L)
```

### tools.R

```
action <- function(x) {
  stopifnot(class(x) == "data.frame")
  invisible(x)
}
```

Then, from the main menu, ensure Debug -> On Error -> Break in Code is set. Finally, try to run the code in `script.R` line-by-line (e.g. using <kbd>Ctrl + Enter</kbd>).

### Describe the problem in detail

RStudio opens the R debugger when the error occurs, but fails to navigate to the location where the error occurred.

### Describe the behavior you expected

RStudio should highlight the code in `tools.R` where the error was emitted.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Notes

The error goes away if we disable the `compiler` JIT; e.g.

```
source("tools.R")
compiler::enableJIT(0)
action(mtcars)
action(1L)
```

so I suspect the compiler's JIT (which byte-compiles closures before they are run) is confusing our debugger.

**Comment by mikebessuille** [comment]
Removing milestone - we will review this if it's happening

**Comment by OfekShilon** [comment]
I think this is solved now, probably along with https://github.com/rstudio/rstudio/issues/6922

**Comment by mikebessuille** [comment]
Fantastic!  I'll close this as fixed in 1.4.
